packages/mediapipe-task-text/analysis_options.yaml

```diff
@@ -0,0 +1,8 @@
+include: package:flutter_lints/flutter.yaml
```

Comment:
Should this whole package just bubble up to one analysis options file in the root for consistency? We do this in flutter/packages.